### PR TITLE
BookB-Pset_GeoScienceFeatureTypeHydroGeoUnit

### PR DESCRIPTION
**Pull request by larswik**
# Pset: Pset_GeoScienceFeatureTypeHydroGeoUnit

## Description: 

Properties for hydrogeo units

| Property | Datatype | Description |
|----------|----------|-------------|
|UnitRecharge|IfcVolumetricFlowRateMeasure|Volumetric flow rate of water that enters an hydrogeologic unit, at potentially multiple locations.|
|UnitDischarge|IfcVolumetricFlowRateMeasure|Volumetric flow rate of water that goes out of an hydrogeologic unit, at potentially multiple locations.|
|PermeabilityHoz|IfcVaporPermeabilityMeasure|Value of horizontal permeability.|
|PermeabilityVert|IfcVaporPermeabilityMeasure|Value of vertical permeability.|
|Transmissivity|IfcReal|Parameter governing the flow of water flowing per unit width of the saturated zone of a continuous aquifer (measured in a direction orthogonal to that of flow), and per unit of hydraulic gradient.|
|StorageCoefficient|IfcPositiveRatioMeasure|Ratio of the volume of water released or stored per unit area of an aquifer, to the corresponding charge in hydraulic head, without reference to time.|

### CODE REVIEW

File: IFC4x3/Properties/p/PermeabilityHoz_0cGG3NyxfEXf7guGaaxUkx/DocProperty.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PermeabilityHoz_0cGG3NyxfEXf7guGaaxUkx" Name="PermeabilityHoz" UniqueId="264100d7-f3ba-4e86-91ea-e10924edebbb" PrimaryDataType="IfcLinearVelocityMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/p/PermeabilityHoz_0cGG3NyxfEXf7guGaaxUkx/DocProperty.xml]: As with other permeability properties, let us not use this measure. Maybe discuss what to do before using this data type.

Also, propose changing property name to "PermeabilityHorizontal".

File: IFC4x3/Properties/p/PermeabilityVert_1aKYiWegb7iPXpy4cjfbOY/DocProperty.xml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PermeabilityVert_1aKYiWegb7iPXpy4cjfbOY" Name="PermeabilityVert" UniqueId="64522b20-a2a9-47b1-9873-f049ada65622" PrimaryDataType="IfcLinearVelocityMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/p/PermeabilityVert_1aKYiWegb7iPXpy4cjfbOY/DocProperty.xml]: Propose name "PermeabilityVertical" and data type of course.

File: IFC4x3/Properties/s/StorageCoefficient_36wybY1$X62RvbVDalUFnE/DocProperty.xml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="StorageCoefficient_36wybY1X62RvbVDalUFnE" Name="StorageCoefficient" UniqueId="c6ebc962-07f8-4609-be65-7cd92f78fc4e" PrimaryDataType="IfcReal" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/s/StorageCoefficient_36wybY1$X62RvbVDalUFnE/DocProperty.xml]: Propose one of the ratio measures.

File: IFC4x3/Properties/t/Transmissivity_1md2GJC9f2MflqkMhIDr4K/Documentation.md [hunk 4]
```diff
@@ -0,0 +1 @@
+Parameter governing the flow of water flowing per unit width of the saturated zone of a continuous aquifer (measured in a direction orthogonal to that of flow), and per unit of hydraulic gradient.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/t/Transmissivity_1md2GJC9f2MflqkMhIDr4K/Documentation.md:1]: Not clear what units of measurement there are.

File: IFC4x3/Properties/t/Transmissivity_1md2GJC9f2MflqkMhIDr4K/DocProperty.xml [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Transmissivity_1md2GJC9f2MflqkMhIDr4K" Name="Transmissivity" UniqueId="709c2413-309a-425a-9bf4-b96ad2375114" PrimaryDataType="IfcReal" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/t/Transmissivity_1md2GJC9f2MflqkMhIDr4K/DocProperty.xml]: Data type is problematic.

File: IFC4x3/Properties/p/PermeabilityHoz_0cGG3NyxfEXf7guGaaxUkx/DocProperty.xml (hunk 1 above)
Comment (larswik) [reply IFC4x3/Properties/p/PermeabilityHoz_0cGG3NyxfEXf7guGaaxUkx/DocProperty.xml]: Regarding naming, I agree! Regarding datatype, discuss with @j-weil.

File: IFC4x3/Properties/p/PermeabilityVert_1aKYiWegb7iPXpy4cjfbOY/DocProperty.xml (hunk 2 above)
Comment (larswik) [reply IFC4x3/Properties/p/PermeabilityVert_1aKYiWegb7iPXpy4cjfbOY/DocProperty.xml]: Same comment as for horizontal.

File: IFC4x3/Properties/t/Transmissivity_1md2GJC9f2MflqkMhIDr4K/Documentation.md (hunk 4 above)
Comment (larswik) [reply IFC4x3/Properties/t/Transmissivity_1md2GJC9f2MflqkMhIDr4K/Documentation.md:1]: Excel states real but I had the same question. Unit in excel is m2/s.

File: IFC4x3/Properties/t/Transmissivity_1md2GJC9f2MflqkMhIDr4K/DocProperty.xml (hunk 5 above)
Comment (larswik) [reply IFC4x3/Properties/t/Transmissivity_1md2GJC9f2MflqkMhIDr4K/DocProperty.xml]: See previous comment.

File: IFC4x3/Properties/s/StorageCoefficient_36wybY1$X62RvbVDalUFnE/DocProperty.xml (hunk 3 above)
Comment (larswik) [reply IFC4x3/Properties/s/StorageCoefficient_36wybY1$X62RvbVDalUFnE/DocProperty.xml]: Yes, it should be. Changed to IfcPositiveRatioMeasure in commit [d533080](https://github.com/bSI-InfraRoom/IFC-Specification/pull/626/commits/d533080c39808834c51d8c2557d8771fa1c508f3).

File: IFC4x3/Properties/p/PermeabilityHoz_0cGG3NyxfEXf7guGaaxUkx/DocProperty.xml (hunk 1 above)
Comment (larswik) [reply IFC4x3/Properties/p/PermeabilityHoz_0cGG3NyxfEXf7guGaaxUkx/DocProperty.xml]: Renamed to HydraulicPermeabilityHorizontal/Vertical. Regarding datatype @j-weil persists in IfcLInearVelocityMeasure. Pset_InSituTestHydro is already merged with a HydraulicPermeability which also had IfcLInearVelocityMeasure.

File: IFC4x3/Properties/p/PermeabilityVert_1aKYiWegb7iPXpy4cjfbOY/DocProperty.xml (hunk 2 above)
Comment (larswik) [reply IFC4x3/Properties/p/PermeabilityVert_1aKYiWegb7iPXpy4cjfbOY/DocProperty.xml]: See response for horizontal.

File: IFC4x3/Properties/t/Transmissivity_1md2GJC9f2MflqkMhIDr4K/Documentation.md (hunk 4 above)
Comment (larswik) [reply IFC4x3/Properties/t/Transmissivity_1md2GJC9f2MflqkMhIDr4K/Documentation.md:1]: Reused Transmissivity from PSet_InSituTestHydro.

File: IFC4x3/Properties/t/Transmissivity_1md2GJC9f2MflqkMhIDr4K/DocProperty.xml (hunk 5 above)
Comment (larswik) [reply IFC4x3/Properties/t/Transmissivity_1md2GJC9f2MflqkMhIDr4K/DocProperty.xml]: Reused Transmissivity from Pset_InSituTestPumping.

**Pull request merged**